conf/test.config

```diff
@@ -108,6 +108,7 @@ if (System.getenv('PROFILE')) {
     } else if ("$PROFILE" == "docker") {
         conda.enabled                            = false
         docker.enabled                           = true
+        docker.runOptions                        = '-u $(id -u):$(id -g) -v /etc/passwd:/etc/passwd:ro -v /etc/shadow:/etc/shadow:ro -v /etc/group:/etc/group:ro'
```

Comment:
Here we have `docker.userEmulation   = { params.use_gatk_spark ? false : true }.call()`  on the master-branch.

Reply:
could consider to only overwrite when running with spark & joint germline and not generally?